app-libs/stf/src/stf_sgx.rs

```diff
@@ -57,33 +73,36 @@ impl Stf {
 		ext
 	}
 
-	pub fn get_state(ext: &mut State, getter: Getter) -> Option<Vec<u8>> {
+	pub fn get_state(ext: &mut impl SgxExternalitiesTrait, getter: Getter) -> Option<Vec<u8>> {
 		ext.execute_with(|| match getter {
 			Getter::trusted(g) => match g.getter {
-				TrustedGetter::free_balance(who) =>
+				TrustedGetter::free_balance(who) => {
```

Comment:
Seems like there was a change in the formatter after some update that happened on master? Now we have some formatting changes in this PR as a result.